completion_aggregator/api/v1/views.py

```diff
@@ -388,3 +390,109 @@ def get(self, request, course_key):
             many=True
         )
         return paginator.get_paginated_response(serializer.data)
+
+
+class CourseLevelCompletionView(CompletionViewMixin, APIView):
```

Comment:
This name is semantically pretty similar to what the CompletionDetailView returns.  Can you include `Stats` in the name, so the distinction is clearer?